cms/djangoapps/contentstore/toggles.py

```diff
@@ -116,6 +116,9 @@ def use_new_text_editor():
 
 
 def use_new_video_editor():
+    # TODO: Remove next 2 lines
+    # uncomment next line to test new video editor redirect
+    # return True
```

Comment:
Ah ok, so https://2u-internal.atlassian.net/wiki/spaces/SRE/pages/19243870/Django+Administration might be a good place to start to demonstrate how waffle flags work. Let me know if you want me to demo that for you at some point (once I teach you about this feature, we won't need this comment)

Reply:
https://github.com/openedx/frontend-app-course-authoring#requirements-1 has the specific waffle flags we use to enable these features.

Reply:
Done